Kite SDK/PSPrintSDK/KitePrintSDK-Info.plist

```diff
@@ -48,8 +48,6 @@
 	<key>UISupportedInterfaceOrientations</key>
 	<array>
 		<string>UIInterfaceOrientationPortrait</string>
-		<string>UIInterfaceOrientationLandscapeLeft</string>
-		<string>UIInterfaceOrientationLandscapeRight</string>
```

Comment:
I added rotation to the demo app so that we can test that none of our screens rotate even if the host app does. Remember how the credit card screen rotated? We probably would have caught that if the demo app could rotate (it didn't at that time)